rollup.config.js

```diff
@@ -1,17 +1,47 @@
+import { nodeResolve } from "@rollup/plugin-node-resolve";
 import commonjs from "@rollup/plugin-commonjs";
+import replace from "@rollup/plugin-replace";
 import { terser } from "rollup-plugin-terser";
 
-function createConfig(format) {
+function createConfig(format, options = {}) {
   const dir = format === "module" ? "esm" : format;
+  const { resolved = false, min = false } = options;
   return {
-    input: require.resolve("autopublish-template"),
+    input: require.resolve("react-emotion/dist/index.esm"),
     output: {
-      file: `${dir}/index.js`,
+      file: `${dir}/index${resolved ? ".resolved" : ""}${min ? ".min" : ""}.js`,
       sourcemap: true,
       format,
+      paths: {
+        react: resolved
+          ? `//cdn.jsdelivr.net/npm/@esm-bundle/react@16.13.1/esm/react.production.min.js`
+          : "react",
+        emotion: resolved
+          ? `//cdn.jsdelivr.net/npm/@esm-bundle/emotion@9.2.12/esm/emotion.min.js`
```

Comment:
Same here about using the latest version of emotion

Reply:
That wouldn't be possible for this import, react-emotion is only compatible with emotion@9. With emotion@10, they've renamed this package to @emotion/core.